prediction_market_agent_tooling/markets/metaculus/data_models.py

```
@@ -3,93 +3,85 @@
-    my_predictions: UserPredictions | None = None
+    question: Question
     # TODO add the rest of the fields https://github.com/gnosis/prediction-market-agent-tooling/issues/301
 
+    @property
+    def page_url(self) -> str:
+        return f"https://www.metaculus.com/questions/{self.id}/"
+
+    @property
+    def p_yes(self) -> Probability:
+        if self.question.type != QuestionType.binary:
+            raise ValueError(f"Only binary markets can have p_yes.")
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)